README.md

```diff
@@ -0,0 +1,76 @@
+# Juttle S3 Adapter
+
+[![Build Status](https://travis-ci.org/juttle/juttle-s3-adapter.svg)](https://travis-ci.org/juttle/s3-adapter)
+
+The Juttle S3 Adapter enables interaction with [S3](https://github.com/juttle/s3).
+
+## Examples
+
+Write a point with value 1 and name `s3_test` to the S3 bucket `my_bucket` with the name `my_point.json`.
+
+```juttle
+emit -limit 1
+| put value = 1, name='s3_test'
+| write s3 -bucket 'my_bucket' -key 'my_point.json'
+```
+Read the file `my_point.json` stored in the S3 bucket `my_bucket`.
+
+```juttle
+read s3 -bucket 'my_bucket' -key 'my_point.json'
+```
+
+## Installation
+
+Like Juttle itself, the adapter is installed as an npm package. Both Juttle and
+the adapter need to be installed side-by-side:
+
+```bash
+$ npm install juttle
+$ npm install juttle-s3-adapter
+```
+
+## Configuration
+
+The adapter needs to be registered and configured so that it can be used from
+within Juttle. To do so, add the following to your `~/.juttle/config.json` file:
+
```

Comment:
What if I'm reading from public s3 bucket, do I add nothing to the config file, or an empty `s3 { }` block?